gax-java/gax/src/main/java/com/google/api/gax/retrying/RetrySettings.java

```diff
@@ -74,36 +78,53 @@ public abstract class RetrySettings implements Serializable {
 
   /**
    * TotalTimeout has ultimate control over how long the logic should keep trying the remote call
-   * until it gives up completely. The higher the total timeout, the more retries can be attempted.
-   * The default value is {@code Duration.ZERO}.
+   * until it gives up completely. The higher the total timeout, the more retries and polls can be
+   * attempted. The default value is {@code Duration.ZERO}.
+   *
+   * <p>If there are no configurations, Retries have the default timeout value set above and LROs
+   * have a default total timeout value of {@code Duration.ofMillis(300000)} (5 minutes).
    */
   public abstract Duration getTotalTimeout();
 
   /**
-   * InitialRetryDelay controls the delay before the first retry. Subsequent retries will use this
-   * value adjusted according to the RetryDelayMultiplier. The default value is {@code
+   * InitialRetryDelay controls the delay before the first retry/ poll. Subsequent retries and polls
+   * will use this value adjusted according to the RetryDelayMultiplier. The default value is {@code
    * Duration.ZERO}.
+   *
+   * <p>If there are no configurations, Retries have the default initial retry delay value set above
+   * and LROs have a default initial poll delay value of {@code Duration.ofMillis(5000)} (5
```

Comment:
Where is the default number coming from? Can we link to where the default number is actually specified? Same comment for all the default numbers below.

Reply:
I have a reference here: https://github.com/googleapis/sdk-platform-java/blob/14973463983495c7eb386a0edfabfcac3fcf9359/gax-java/gax/src/main/java/com/google/api/gax/retrying/RetrySettings.java#L70-L72, but it's at the top of the file and may get lost. The default value retry values are populated from the grpc config file and I don't want to link those in here. LROs are preconfigured from the generation.

Reply:
What are the default numbers if there is no gRPC config file? I guess the numbers are coming from constants in GAX ? The concern I have is if we change the default numbers in the future, we would have to change the numbers here as well, and it's very easy to forget to change here.

Reply:
If there is nothing configured inside the grpc config file, the generator will create a RetrySettings with the default values with values listed above (For example: https://github.com/googleapis/google-cloud-java/blob/8452ac3a8eefa8a9e49ff83187b33be26efe6a01/java-batch/google-cloud-batch/src/main/java/com/google/cloud/batch/v1/stub/BatchServiceStubSettings.java#L518). This is what I'll call the RetrySettings default values.

Service teams can modify the grpc config file and that will provide default values for certain callables. These default values is what I'll call the callable default values. This is the RetrySettings which will be configured like: https://github.com/googleapis/google-cloud-java/blob/8452ac3a8eefa8a9e49ff83187b33be26efe6a01/java-batch/google-cloud-batch/src/main/java/com/google/cloud/batch/v1/stub/BatchServiceStubSettings.java#L507-L516

I believe they're both technically default values. The Javadoc I updated only refers the RetrySettings default values since the callable default values differ per service. I can add this explanation in so it's clearer to the readers.

Reply:
The default numbers in [RetrySettingsComposer](https://github.com/googleapis/sdk-platform-java/blob/main/gapic-generator-java/src/main/java/com/google/api/generator/gapic/composer/common/RetrySettingsComposer.java#L69-L73) is what I was referring to. If those numbers change, we would have to update the numbers here as well. But since those numbers are in the generator, I don't think it's a good idea to expose the implementation details to Gax either, I think we can leave them as they are for now.

Reply:
> The default numbers in [RetrySettingsComposer](https://github.com/googleapis/sdk-platform-java/blob/main/gapic-generator-java/src/main/java/com/google/api/generator/gapic/composer/common/RetrySettingsComposer.java#L69-L73) is what I was referring to. If those numbers change, we would have to update the numbers here as well. But since those numbers are in the generator, I don't think it's a good idea to expose the implementation details to Gax either, I think we can leave them as they are for now.

Yeah, those values are set in the generator (and only the generator sets defaults for LROs). I think if we make an effort to change those, we'll have to remember to update our docs here manually. I'd like for an automated way to do that, but linking to the generator code doesn't seem ideal.

I'll keep as is for now then.